javascript/javascript1/week1/housePrice.js

```diff
@@ -0,0 +1,44 @@
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300
+
+// perters parameters 
+var wide = 8 ;
+var deep = 10;
+var height = 10;
+var gardinSize = 100;
```

Comment:
you can use `let` for these variables and later for Julia you can assign new values like this
```js
// julia parameters
wide = 5;
deep = 11;
height = 8;
gardinSize = 70;
```